docker-compose.yml

```diff
@@ -230,3 +230,21 @@ services:
       - ./config/sparql-authorization-wrapper:/config
     restart: always
     logging: *default-logging
+  deliver-email:
+    image: redpencil/deliver-email-service:0.3.3
+    profiles:
+      - send-emails
+    environment:
+      MAILBOX_URI: 'http://data.lblod.info/id/mailboxes/1'
+      SECURE_CONNECTION: "true"
+      EMAIL_PROTOCOL: "smtp"
+      EMAIL_ADDRESS: "mymail@myservice.com"
+      EMAIL_PASSWORD: "myemailpassword"
+      FROM_NAME: "Lokaal Mandatenbeheer"
```

Comment:
Create ticket for Tim fore these values

Reply:
None of these are set in this project (https://github.com/lblod/app-complaint-form/blob/6ec0880bac51e5b0a2aaa60af784d42de7f06aad/docker-compose.yml#L114) this is done in the pipeline?

Reply:
Checked with Achraf: "rechtstreeks op server. Worden gewoon in database opgeslagen. Geen injecties of pipelines of dergelijke"